tests/phpunit/tests/rest-api/rest-categories-controller.php

```diff
@@ -1236,4 +1254,91 @@ protected function check_get_taxonomy_term_response( $response ) {
 		$category = get_term( 1, 'category' );
 		$this->check_taxonomy_term( $category, $data, $response->get_links() );
 	}
+
+	/**
+	 * @dataProvider data_readable_http_methods
+	 * @ticket 56481
+	 *
+	 * @param string $method HTTP method to use.
+	 */
+	public function test_get_items_returns_only_fetches_ids_for_head_requests( $method ) {
```

Comment:
```suggestion
	public function test_get_items_returns_only_ids_for_head_requests( $method ) {
```
For this and other same-named tests in the updated test classes under this PR, `returns_only_fetches` has two verbs, but should only use one for clarity.

Scanning other test classes, "return" is used more extensively than "fetch", so IMHO these new methods should use `returns_only_ids`. What do you think?

Reply:
Well spotted, @ironprogrammer! Having two verbs here doesn't make sense. This is likely the result of extensive copy-pasting between different test classes. :)

I'm leaning towards `test_get_items_fetches_only_ids_for_head_requests`, because what I meant is that the `get_items()` method fetches only the `*id*` column from the database for HEAD requests, as opposed to fetching all columns for non-HEAD requests.

Fixed in 9c91e455805d50251480a465faf0c499e4f37692.